### PR TITLE
Add Heroku update script for easy management of Heroku updates

### PR DESCRIPTION
**Pull request by JL2210**
Oh no! @skiman6010's worst fear!

This package pulls the latest tag form the heroku/cli git repository and then creates a functional Chromebrew package from there.

### CODE REVIEW

File: tools/heroku.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+export PWD_HOLD="${PWD}"
```
Comment (uberhacker) [inline tools/heroku.sh]: No need to export if this is used only in this script.

File: tools/heroku.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+echo >> heroku.rb
+echo "  def self.install" >> heroku.rb
+echo "    system \"mkdir -p #{CREW_DEST_PREFIX}/share/heroku\"" >> heroku.rb
+echo "    system \"cp -r . #{CREW_DEST_PREFIX}/share/heroku\"" >> heroku.rb
+echo "    system \"mkdir -p #{CREW_DEST_PREFIX}/bin\"" >> heroku.rb
+echo "    system \"ln -s #{CREW_PREFIX}/share/heroku/bin/run #{CREW_DEST_PREFIX}/bin/heroku\"" >> heroku.rb
+echo "  end" >> heroku.rb
+echo "end" >> heroku.rb
+
+unset PWD_HOLD
```
Comment (uberhacker) [inline tools/heroku.sh]: No need if not exported.

File: tools/heroku.sh (hunk 1 above)
Comment (JL2210) [reply tools/heroku.sh]: See below.

**Comment by uberhacker** [comment]
This is a nice idea but what would make it even cooler is if you could create the binaries and upload to bintray.com automagically.  If you could do this, the entire build process could be automated.  It might be even better if the script was general purpose for any package.

File: tools/heroku.sh (hunk 2 above)
Comment (JL2210) [reply tools/heroku.sh]: Also, it only affects newly formed child processes.

**Comment by JL2210** [comment]
That would be extremely difficult. Right now, I don't see it being possible for any other package, and even if it was, it would take a lot of hand-written scripts. About the binaries, the script would require people to have 4 VMs with Chromium OS installed to work. I gather that that's much too involved for even the average maintainer.

**Comment by JL2210** [comment]
The new scripts would also require a Git repository and marked releases to work. Not very portable, if I do say so myself.

File: tools/heroku.sh (hunk 2 above)
Comment (uberhacker) [reply tools/heroku.sh]: Any variable you assign will be available in the script without the need to export.  Not sure what weird shenanigans you are talking about but every script I have ever written did not need exported variables.

Comment (JL2210) [reply tools/heroku.sh]: OK.

Comment (JL2210) [reply tools/heroku.sh]: I'm not sure about that anymore. I couldn't reproduce it today.

Comment (JL2210) [reply tools/heroku.sh]: Done.

File: tools/heroku.sh (hunk 1 above)
Comment (JL2210) [reply tools/heroku.sh]: Done.

**Comment by JL2210** [comment]
@uberhacker, this OK?

**Comment by uberhacker** [comment]
I'm passing the baton to @skiman6010. :)

**Comment by skiman6010** [comment]
Cool script! Thanks!

**Pull request merged**